### PR TITLE
CIAC-11004 Feeds Enrichment Excluded field

### PR DESCRIPTION
**Pull request by itssapir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-11004](https://jira-dc.paloaltonetworks.com/browse/CIAC-11004)
## Description
Support for settings select feeds as 'Enrichment excluded', which sets all of the indicators fetched by the instance to enrichment excluded.

Feeds supported by this PR:
- AzureFeed
- Office 365 Feed
- Cisco WebEx Feed
- Cloudflare Feed
- Fastly Feed
- AWS Feed
- Zoom Feed
- Public DNS Feed
- Google IP Ranges Feed

Updated the Collect/Connect fields

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bziser — APPROVED** [review]
looks great!!!

File: Packs/FeedFastly/ReleaseNotes/1_1_29.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Fastly Feed
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion
```
Comment (ShirleyDenkberg) [inline Packs/FeedFastly/ReleaseNotes/1_1_29.md:7]:
```suggestion
- Support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedGCPWhitelist/Integrations/FeedGoogleIPRanges/README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Use the Google IP Ranges Feed integration to get GCP and Google global IP ranges
 | feedFetchInterval | Feed Fetch Interval | False |
 | feedTags | Tags | False |
 | feedBypassExclusionList | Bypass exclusion list | False |
+| Enrichment Excluded | Select this option to exclude the fetched indicators from the enrichment process | False |
```
Comment (ShirleyDenkberg) [inline Packs/FeedGCPWhitelist/Integrations/FeedGoogleIPRanges/README.md:21]:
```suggestion
| Enrichment Excluded | Select this option to exclude the fetched indicators from the enrichment process. | False |
```

File: Packs/FeedOffice365/Integrations/FeedOffice365/README.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ The Office 365 IP Address and URL web service is a read-only API provided by Mic
    | feedExpirationInterval |  |  |
    | Feed Fetch Interval | How often to fetch indicators from this integration instance. You can specify the interval in days, hours, or minutes. | 30 minutes |
    | Bypass exclusion list | When selected, the exclusion list is ignored for indicators from this feed.  This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system. | N/A |
+    | Enrichment Excluded | Select this option to exclude the fetched indicators from the enrichment process | False |
```
Comment (ShirleyDenkberg) [inline Packs/FeedOffice365/Integrations/FeedOffice365/README.md:24]:
```suggestion
    | Enrichment Excluded | Select this option to exclude the fetched indicators from the enrichment process. | False |
```

File: Packs/FeedOffice365/ReleaseNotes/1_2_13.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Office 365 Feed
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion
```
Comment (ShirleyDenkberg) [inline Packs/FeedOffice365/ReleaseNotes/1_2_13.md:7]:
```suggestion
- Support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedPublicDNS/Integrations/FeedPublicDNS/FeedPublicDNS.yml [hunk 5]
```diff
@@ -47,44 +54,68 @@ configuration:
...
   required: false
-- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed
-  display: Traffic Light Protocol Color
+  section: Collect
+  advanced: true
+- display: Traffic Light Protocol Color
   name: tlp_color
+  type: 15
+  required: false
+  additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed
```
Comment (ShirleyDenkberg) [inline Packs/FeedPublicDNS/Integrations/FeedPublicDNS/FeedPublicDNS.yml:91]:
```suggestion
  additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed.
```

File: Packs/FeedPublicDNS/Integrations/FeedPublicDNS/README.md [hunk 6]
```diff
@@ -17,6 +17,7 @@ A feed of known benign IPs of public DNS servers.
     | feedTags | Tags | False |
     | feedBypassExclusionList | Bypass exclusion list | False |
     | tlp_color | Traffic Light Protocol Color | False |
+    | Enrichment Excluded | Select this option to exclude the fetched indicators from the enrichment process | False |
```
Comment (ShirleyDenkberg) [inline Packs/FeedPublicDNS/Integrations/FeedPublicDNS/README.md:20]:
```suggestion
    | Enrichment Excluded | Select this option to exclude the fetched indicators from the enrichment process. | False |
```

File: Packs/FeedSpamhaus/ReleaseNotes/1_1_24.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Spamhaus Feed
+
+- Enhanced the ***HTTPFeedApiModule*** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedSpamhaus/ReleaseNotes/1_1_24.md:6]:
```suggestion
Enhanced **HTTPFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedZoom/Integrations/FeedZoom/FeedZoom.yml [hunk 8]
```diff
@@ -81,37 +92,60 @@ configuration:
...
 - display: Bypass exclusion list
   name: feedBypassExclusionList
   defaultvalue: "true"
   type: 8
   additionalinfo: When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system.
   required: false
+  section: Collect
+  advanced: true
+- display: Enrichment Excluded
+  additionalinfo: Select this option to exclude the fetched indicators from the enrichment process
```
Comment (ShirleyDenkberg) [inline Packs/FeedZoom/Integrations/FeedZoom/FeedZoom.yml:129]:
```suggestion
  additionalinfo: Select this option to exclude the fetched indicators from the enrichment process.
```

File: Packs/FeedZoom/ReleaseNotes/1_1_15.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Zoom Feed
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion
```
Comment (ShirleyDenkberg) [inline Packs/FeedZoom/ReleaseNotes/1_1_15.md:7]:
```suggestion
- Support for instance-wide indicator enrichment exclusion.
```

**Comment by ShirleyDenkberg** [comment]
@bziser Doc review completed.

**Comment by itssapir** [comment]
Need to force merge due to:
- Checks require updating deprecated packs - [CIAC-11618](https://jira-dc.paloaltonetworks.com/browse/CIAC-11618)
- Zoom TPB fails due to bug with dependency loading to test instance - [CIAC-9721](https://jira-dc.paloaltonetworks.com/browse/CIAC-9721)

**Pull request merged**